libs/libarchfpga/src/physical_types.h

```diff
@@ -611,6 +606,8 @@ struct t_physical_tile_type {
 
     int index = -1; /* index of type descriptor in array (allows for index referencing) */
 
+    std::vector<t_sub_tile> sub_tiles;
+
     std::vector<t_logical_block_type_ptr> equivalent_sites;
```

Comment:
Probably on the TODO list, but it would seem that this should be removed since it's now in the sub-tiles.